test_wazuh/test_fim/test_basic_usage/test_basic_usage_move_dir.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) 2015-2019, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import shutil
+import pytest
+from wazuh_testing.fim import LOG_FILE_PATH, generate_params, create_file, REGULAR, \
+    callback_detect_event, check_time_travel, DEFAULT_TIMEOUT
+from wazuh_testing.tools import FileMonitor, check_apply_test, load_wazuh_configurations, PREFIX
+
+# variables
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+test_directories = [os.path.join(PREFIX, 'testdir1'), os.path.join(PREFIX, 'testdir2'),
```

Comment:
I would include `testdir4` in `test_directories` variable to allow the fixture deleting it after running the test. In consequence, the `directory_str` should be updated to exclude the `testdir4`.

Reply:
Ok, I'm wrong. If you include `testdir4` in `test_directories` the folder could be created in advance, which would break the test. Never mind :)